src/xai_grok/__init__.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+import requests
+
+from xai_grok.grok import Grok as GrokClient
+import xai_grok.providers as providers
+
+
+def Grok(api_key: str = ""):
+    user_api_key = api_key if api_key else os.environ["XAI_API_KEY"]
```

Comment:
The code will raise a `KeyError` if the `XAI_API_KEY` environment variable is not set and no API key is provided. Should handle this case gracefully with a clear error message.